ported_projects.md

```diff
@@ -0,0 +1,12 @@
+# New and Ported Projects
```

Comment:
It might be good to call out where in the lifecycle these ports are.

Reply:
Good point.  Will do.